lib/experimental/subject.rb

```diff
@@ -12,8 +12,8 @@ def experiment_bucket(name)
       Experimental.source[name].try { |e| e.in?(self) ? e.bucket(self) : nil }
     end
 
-    def in_bucket?(name, bucket)
-      in_experiment?(name) && experiment_bucket(name) == bucket
+    def in_bucket?(name, bucket, *args)
```

Comment:
I see your point.  With the above, are you more inclined to have the user pass an array?  I guess my only concern with that approach is having a parameter of 2 types.  We could alternatively `args.flatten << bucket` which would take care of other junk (though, I think junk arguments like that should throw)